pkg/metrics/grafana/tidb_resource_control.json

```diff
@@ -223,7 +223,7 @@
           "targets": [
             {
               "editorMode": "code",
-              "expr": "sum(resource_manager_resource_unit_read_request_unit_max_per_sec{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\"}) by (resource_group)",
+              "expr": "sum(resource_manager_resource_unit_read_request_unit_max_per_sec{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\"}) by (resource_group) + sum(resource_manager_resource_unit_write_request_unit_max_per_sec{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\"}) by (resource_group)",
```

Comment:
We can display the type separately. The maximum is the highest RU in the last 20 seconds. These may not occur simultaneously, so it's better to show them separately.

Reply:
fixed